android/app/src/main/java/org/uwaterloo/subletr/pages/chat/ChatListingPageViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.uwaterloo.subletr.pages.chat
+
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import io.reactivex.rxjava3.subjects.BehaviorSubject
+import org.uwaterloo.subletr.R
+import org.uwaterloo.subletr.api.apis.DefaultApi
+import org.uwaterloo.subletr.enums.Gender
+import org.uwaterloo.subletr.pages.account.AccountPageUiState
+import org.uwaterloo.subletr.services.INavigationService
+import javax.inject.Inject
+
+@HiltViewModel
+class ChatListingPageViewModel @Inject constructor(
+	navigationService: INavigationService,
+	defaultApi: DefaultApi,
+) : ViewModel() {
+	val uiStateStream: BehaviorSubject<ChatListingPageUiState> = BehaviorSubject.createDefault(
+		ChatListingPageUiState.Loaded(
+			contacts = emptyList()
+		)
+	)
+}
+
+class Contact(
```

Comment:
Do you want a data class here?

Reply:
Sorry, that wasn't really a question.
You should probably use a data class here.

Reply:
Data Class plz